packages/pluggableWidgets/video-player-native/src/ui/Styles.ts

```diff
@@ -26,5 +30,28 @@ export const defaultVideoStyle: VideoStyle = {
     },
     errorMessage: {
         color: "white"
+    },
+    fullScreenVideoPlayer: {
```

Comment:
These styles are new. Best to adjust [Mendix styling documentation](https://docs.mendix.com/refguide/native-styling-refguide#11-18-video-player) and Atlas.

Atlas: maybe update just with property keys and empty object `{}` values. By adding the properties, when end-users open the styling file in IDE or Studio Pro, they will at least habe all the different "sections" for styling. Please let me know if this doesnt make sense to you.

Reply:
done

Reply:
done